src/content/contributing/release-process/_index.en.md

```diff
@@ -0,0 +1,146 @@
+
+For this example the build can be found [here](https://travis-ci.com/github/submariner-io/lighthouse/builds/153946391).
+
+Look under the "Active branches" section for v0.2.0-rc0, and monitor the build.
+Verify that the build successfully completes as indicated by a green checkmark at the right. At this point the images are pushed to quay.io
+
+> https://quay.io/repository/submariner/lighthouse-controller?tab=tags
+> https://quay.io/repository/submariner/lighthouse-coredns?tab=tags
+
+<!-- TODO: Aswin: the openshift-coredns part -->
+
+# Step 3: update the operator version references and create a release
+
+Once the other builds have finished and you have 0.2.0-rc0 release tags for the submariner and lighthouse projects, you can proceed with changes to the operator.
+
+## Change referenced versions
+
+Edit the operator [versions](https://github.com/submariner-io/submariner-operator/edit/master/pkg/versions/versions.go) file and change the project version constants to reference the new release, "0.2.0-rc0".
+
+> https://github.com/submariner-io/submariner-operator/edit/master/pkg/versions/versions.go
+
+Create a pull request, wait for the CI job to pass, and get approval/merge. See an example PR [here](https://github.com/submariner-io/submariner-operator/pull/276)
+
+
+## Create a submariner-operator release
+
+Assuming you have an existing submariner-operator git directory, run the following steps:
+
+```bash
+cd submariner-operator
+git remote add upstream ssh://git@github.com/submariner-io/submariner-operator
+git fetch -a -v -t
+git checkout remotes/upstream/master -B master
+git tag -s -m "Globalnet Overlapping IP support RC0" v0.2.0-rc0
+git push upstream v0.2.0-rc0
+```
+
+
+A tagged release should appear [here](https://github.com/submariner-io/submariner-operator/tags).
+
+> https://github.com/submariner-io/submariner-operator/tags
+
+A build for v0.2.0-rc0 should start and appear under the under the "Active branches" section [here](https://travis-ci.com/github/submariner-io/submariner-operator/branches).
+
+> https://travis-ci.com/github/submariner-io/submariner-operator/branches
+
+Look under the "Active branches" section for v0.2.0-rc0 , and monitor the build.
+Verify that the build successfully completes as indicated by a green checkmark at the right.
+At this point the images are pushed to quay.io.
+
+Finally once that has finished, a 0.2.0-rc0 tag will be available [here](https://quay.io/repository/submariner/submariner-operator?tab=tags).
+
+> https://quay.io/repository/submariner/submariner-operator?tab=tags
+
+
+## Create the subctl binaries release
+
+
+```bash
+cd submariner-operator
+git fetch -a -v -t
+git checkout v0.2.0-rc0
+rm bin/subctl*
+make build-cross
+ls -la bin/subctl*
+```
+
+Go to https://github.com/submariner-io/submariner-operator/tags , and create a release for the v0.2.0-rc0 tag, upload the generated binaries.
+
+If it's a pre-release (-rc) mark the checkbox "This is a pre-release"
```

Comment:
```suggestion
If this is a pre-release, mark the checkbox "This is a pre-release".
```

Reply:
Technically we wouldn't need the ", " after pre-release, right? or is it necessary? (ignorance asking here)